File: LeetCode/Easy/0832-flipping-an-image/0832-flipping-an-image.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        for (vector<int>& row : image){
            int i = 0;
            int j = row.size() - 1;
            while (i <= j){
                int temp = row[i] == 1 ? 0 : 1;
                row[i] = row[j] == 1 ? 0 : 1;
                row[j] = temp;
                ++i;
                j--;
            }
        }
        
        return image;
    }
};

File: LeetCode/Easy/0832-flipping-an-image/0832-flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        for row in image:
            i, j = 0, len(row) - 1
            
            while i <= j:
                row[i], row[j] = int(not row[j]), int(not row[i])
                i += 1
                j -= 1
        
        return image